src/FirePokemon.java

```diff
@@ -0,0 +1,60 @@
+public class FirePokemon extends Pokemon {
+    private final String TYPE = "Fire \uD83D\uDD25";
+    private String attack;
```

Comment:
Dit komt bij elke pokemon terug, dan had het net zo goed in de super klassen kunnen staan. |

Wat hier wel had kunnen staan is bijvoorbeeld int FireTemp; waardoor je methode met fireTemperature dynamisch had kunnen zijn.